railties/lib/rails/application/configuration.rb

```diff
@@ -500,6 +500,26 @@ def colorize_logging=(val)
         generators.colorize_logging = val
       end
 
+      def secret_key_base
+        @secret_key_base || begin
+          self.secret_key_base = if Rails.env.local? || ENV["SECRET_KEY_BASE_DUMMY"]
+            generate_local_secret
+          else
+            ENV["SECRET_KEY_BASE"] || Rails.application.credentials.secret_key_base
```

Comment:
Is `credentials` already available as an accessor?
```suggestion
            ENV["SECRET_KEY_BASE"] || credentials.secret_key_base
```

Reply:
I initially had the same thought. There is a `credentials` accessor  here, however its for configuration and not the actual `credentials` object on `Rails.application`

https://guides.rubyonrails.org/configuring.html#config-credentials-content-path